iOS/ViewController.m

```diff
@@ -1001,6 +1001,14 @@ -(void) resetView {
     [self dismissNodeInfoPopover];
     [self.controller deselectCurrentNode];
     [self resetVisualization];
+
+    if(self.arEnabled && self.afterViewReset) {
+        // In AR mode we don't actually change the rotation on a view reset, so if there is a post reset callback,
+        // call it now
+        // TODO: should really find a better way to handle this
+        self.afterViewReset();
+        self.afterViewReset = nil;
```

Comment:
There seems to be similar logic in `viewResetDone`. It does a slight delay before invoking the block. Not sure if it matters or not.